test/create-run-publish-app.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
@friism recently made a [contribution](https://github.com/dotnet/dotnet-docker/commit/6ce7bf25a74f8097e18757b484b4cd2c9399302c) to the dotnet-docker repo last week which changed this pattern to use a Dockerfile instead.  This removed the host dependency which allows the tests to be run against a remote daemon.